src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -426,42 +431,44 @@ private AIState requestMaterials()
     private boolean isBlockFree(Block block, int metadata)
     {
         return block == null
-                || BlockUtils.isWater(block.getDefaultState())
-                || block.equals(Blocks.leaves)
-                || block.equals(Blocks.leaves2)
-                || block.equals(Blocks.double_plant) && Utils.testFlag(metadata, 0x08)
-                || block instanceof BlockDoor && Utils.testFlag(metadata, 0x08)
-                || block.equals(Blocks.grass)
-                || block.equals(Blocks.dirt);
+               || BlockUtils.isWater(block.getDefaultState())
+               || block.equals(Blocks.leaves)
+               || block.equals(Blocks.leaves2)
+               || (block.equals(Blocks.double_plant) && Utils.testFlag(metadata, 0x08))
+               || (block instanceof BlockDoor && Utils.testFlag(metadata, 0x08))
+               || block.equals(Blocks.grass)
+               || block.equals(Blocks.dirt);
     }
 
     private AIState structureStep()
     {
-        if(!goToConstructionSite())
+        if (!goToConstructionSite())
         {
             return this.getState();
         }
 
-        if(job.getSchematic().getBlock() == null
-                || job.getSchematic().doesSchematicBlockEqualWorldBlock()
-                || (!job.getSchematic().getBlock().getMaterial().isSolid() && job.getSchematic().getBlock() != Blocks.air))
+        if (job.getSchematic().getBlock() == null
+            || job.getSchematic().doesSchematicBlockEqualWorldBlock()
+            || (!job.getSchematic().getBlock().getMaterial().isSolid()
+                && job.getSchematic().getBlock() != Blocks.air))
         {
-            return findNextBlockSolid();//findNextBlock count was reached and we can ignore this block
+            //findNextBlock count was reached and we can ignore this block
+            return findNextBlockSolid();
         }
 
         worker.faceBlock(job.getSchematic().getBlockPosition());
-        Block block = job.getSchematic().getBlock();
+        Block       block    = job.getSchematic().getBlock();
         IBlockState metadata = job.getSchematic().getBlockState();
 
         BlockPos coordinates = job.getSchematic().getBlockPosition();
-        int x = coordinates.getX();
-        int y = coordinates.getY();
-        int z = coordinates.getZ();
+        int      x           = coordinates.getX();
+        int      y           = coordinates.getY();
+        int      z           = coordinates.getZ();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'z' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)